examples/models/image_object_detection/food_detection/food101.py

```diff
@@ -1,4 +1,4 @@
-from singa_auto.darknet.food_objection_base_model import FoodDetectionBase
+from examples.models.image_object_detection.food_detection.food_objection_base_model import FoodDetectionBase
```

Comment:
do you need the full path to import the module if all files are in the same folder?

Reply:
This is just an example code, If we deploy this app to the singa-auto, we may need to use another way.